### ISSUE TITLE
Simple videos need place to host them

### ISSUE THREAD

**Issue by camusamta** [w3c/wcag#768]
https://www.w3.org/WAI/WCAG21/Understanding/reflow.html#examples is missing https://alastairc.ac/w3c/reflow-example-1-BBC.mp4.

**Comment by awkawk** [comment]
@alastc we should put that video on youtube or somewhere public rather than your personal server.

**Comment by alastc** [comment]
Hmm, I updated the URL for that a while ago, I guess it needs a publishing run.

Agreed that it should be hosted somewhere else, but I'd rather have a place for static files rather than embed a youtube video. We also don't want to start putting video content in the git repo, that would expand it massively.

Is there a static file folder somewhere in the W3C system that could be used for long term, static file storage? (@michael-n-cooper)

**Comment by awkawk** [comment]
See https://www.w3.org/WAI/videos/standards-and-benefits/

**Comment by alastc** [comment]
Yea, I did look at that and (for the video we have) it would have taken a day or two to implement. I didn't have a day. Maybe something we can assign to Hidde?

**Comment by alastc** [comment]
The URL was updated and it's been published.

Still looking for a place for video files, but I'll mark that as "design and templates" and pass to @hidde :-)

**Comment by fstrr** [comment]
@kfranqueiro now that we have hosted videos in the big reflow update, can this be closed?

**Comment by kfranqueiro** [comment]
> [@kfranqueiro](https://github.com/kfranqueiro) now that we have hosted videos in the big reflow update, can this be closed?

No; this has not actually been resolved yet. The reflow PR currently adds them to the repo, but that is not their final resting place. (The question has come up multiple weeks in a row in the backlog TF, and W3C staff will follow up)

**Comment by patrickhlauke** [comment]
might have been mentioned previously, but ... github-lfs worth looking at?

**Comment by kfranqueiro** [comment]
I don't think git-lfs will be tenable for this repo, since it relies on a gh-pages branch, and that does not support LFS. So we'd end up with the same problem specifically on that branch when the content is built, which would still lead to the same overall repository size issue.
